ovos_plugin_manager/templates/tts.py

```diff
@@ -46,16 +46,6 @@
 
 EMPTY_PLAYBACK_QUEUE_TUPLE = (None, None, None, None, None)
 
-
-# TODO move to ovos_utils
-def get_ram_directory(folder):
```

Comment:
instead of removing this, can you add a check for platform inside this method and only use temp if it's windows?

the advantage here is to reduce writes to disk and sd card failures, this file is written to a lot of times, the reasoning here was to extend the lifetime of the sdcard in raspberry pi, which are arguably the most common platform